tests/unit/multiclass/KNN_unittest.cc

```diff
@@ -0,0 +1,49 @@
+#include <shogun/labels/MulticlassLabels.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/multiclass/KNN.h>
+#include <shogun/distance/EuclideanDistance.h>
+#include <gtest/gtest.h>
+#include <shogun/labels/BinaryLabels.h>
+
+using namespace shogun;
+
+TEST (KNN, simple_classification)
```

Comment:
Could you suggest a better name? I'll get an idea of how tests should be named then.